txpipe/source_selector.py

```diff
@@ -1,13 +1,43 @@
 from .base_stage import PipelineStage
 from .data_types import ShearCatalog, YamlFile, PhotozPDFFile, TomographyCatalog, HDFFile, TextFile
-from .utils import SourceNumberDensityStats
+from .utils import SourceNumberDensityStats, rename_iterated
 from .utils.calibration_tools import read_shear_catalog_type, apply_metacal_response
-from .utils.calibration_tools import metacal_variants, band_variants, MetacalCalculator, LensfitCalculator, HSCCalculator
+from .utils.calibration_tools import metacal_variants, metadetect_variants, band_variants, MetacalCalculator, LensfitCalculator, HSCCalculator, MetaDetectCalculator
+from .utils.calibrators import MetaCalibrator, MetaDetectCalibrator, LensfitCalibrator, HSCCalibrator
```

Comment:
this is kind of random, but I wonder if HSCCalibrator should be HSMCalibrator (named after the shear measurement method)

Reply:
In general I would agree, but I think there's a bunch of ways you might calibrate HSM. Whereas it's mainly only the one relevant version of the lensfit calibration.